doc/source/conf.py

```diff
@@ -7,12 +7,12 @@
 # obtain one at https://mozilla.org/MPL/2.0/.
 
 from __future__ import print_function
-from distutils.version import StrictVersion
 import sphinx
 import glob
 import os
 import sphinx_rtd_theme
 import sys
+from pkg_resources import parse_version
```

Comment:
`odl.util` or `parse_version`?

Reply:
Here I think it's better to go for the standard library, since we have a potentially failing import of `odl` further down. It's only for documentation build anyway.